src/components/Home/Aditya/Week3/components/status/statusStyles.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const StatusDiv = styled.div`
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 400px;
+  background-color: #ffe4c4;
+  border-radius: 20px;
+  padding: 20px;
+  box-shadow:
+    inset 10px 10px 10px #87ceeb,
+    inset -10px -10px 10px #e9967a;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
check formatting

Reply:
ES lint is doing this on its own, I cannot change it